pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -86,11 +86,17 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		// If we don't do this, daemons will never start, waiting forever for this configmap to be present
 		//
 		// Only do this when doing a bit of management...
-		logger.Info("creating 'rook-ceph-config' configmap.")
-		err = populateConfigOverrideConfigMap(cluster.context, c.namespacedName.Namespace, cluster.ownerRef)
+		logger.Infof("creating %q configmap", config.StoreName)
```

Comment:
Is StoreName the actual name of the configmap? Seems like it's different.

Reply:
The confusion could come that in many other places in the code, `configmap` is used instead of `secret`. But I might be wrong (see my comment)

Reply:
let me double-check